version.config

```diff
@@ -1,3 +1,3 @@
-version = 2.17.0-dev
+version = 2.17.0
 update_from_version = 2.16.1
 downgrade_to_version = 2.16.1
```

Comment:
```suggestion
version = 2.18.0-dev
update_from_version = 2.17.0
downgrade_to_version = 2.16.1
```

Reply:
This goes into main so the new version for main will be 2.18.0-dev

Reply:
Edited so.